entities/src/main/java/org/odk/collect/entities/javarosa/intance/LocalEntitiesInstanceAdapter.kt

```diff
@@ -23,7 +23,7 @@ class LocalEntitiesInstanceAdapter(private val entitiesRepository: EntitiesRepos
 
                 0.until(count).map {
                     if (it == 0) {
-                        convertToElement(first, true)
+                        convertToElement(first)
                     } else {
                         TreeElement("item", it, true)
```

Comment:
Now I wonder if we need that `isPartial` flag at all... such elements should have at least `label` and `value` right so if they don't have it we could treat them as partial and try to populate.

Reply:
The partial concept is currently too general for something like that as any `TreeElement` can be "partial". It's not confined to secondary instances with `item` children.